src/System.Security.Cryptography.Encryption.Aes/src/Internal/Cryptography/AesImplementation.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal sealed partial class AesImplementation : Aes
+    {
+        public sealed override ICryptoTransform CreateDecryptor(byte[] rgbKey, byte[] rgbIV)
+        {
+            return CreateTransform(rgbKey, rgbIV, encrypting: false);
+        }
+
+        public sealed override ICryptoTransform CreateEncryptor(byte[] rgbKey, byte[] rgbIV)
+        {
+            return CreateTransform(rgbKey, rgbIV, encrypting: true);
+        }
+
+        public sealed override void GenerateIV()
+        {
+            byte[] iv = new byte[this.BlockSize / 8];
```

Comment:
I assume the 8 here is bits-per-byte?  Might be nice to have it in a named constant.